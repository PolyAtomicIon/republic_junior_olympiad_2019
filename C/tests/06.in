12 5
189 245 304 96 465 267 572 914 435 405 789 33 