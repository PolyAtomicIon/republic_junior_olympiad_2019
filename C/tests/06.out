189 1190 4364 11906 27310 55636 104082 182898 305821 490480 759185 1138960 